src/core/CellStructure.hpp

```diff
@@ -663,6 +663,61 @@ struct CellStructure {
 
     return particle_to_cell(p);
   }
-};
 
+public:
+  /**
+   * @brief Run kernel on all particles inside cell and neighbors.
+   *
+   * @param p Particle to identify cell and neighbors
+   * @param kernel Function with signature <tt>double(Particle, Particle,
+   * Vector)</tt>
+   * @return false if cell is not found via @ref particle_to_cell, otherwise
```

Comment:
```suggestion
   * @return false if cell is not found via @c particle_to_cell, otherwise
```
sorry, I forgot that private methods couldn't be referenced in Doxygen